Information/Design.md

```diff
@@ -89,3 +89,320 @@
     - Summary: The documentation shows this REST API should provide all the features we require 
       and is publicly accessible. Using the already provided API and hosting, this option would require less resources 
       than if we were to run our own REST API backend. At this time, this is the suggested backend.
+
+
+
+
+#Design.md by Joshua Rupp
+
+######CEG 4110 Spring 2021 
+
+######Joshua Rupp, Kyle Sturdevant, Trenton Brown --> Group 12
+
+######Overview of what you will find in this document:
+
+[Five requirements of Group 12's Google Doc which can be found at:]
+ 
+(https://docs.google.com/document/d/1LdCnvcZ_LXHHckbBnyPZtQZOH8F8qjPd47igwdDOoVs/edit)
+
+There are 18 total requirements for this project for building and implementing a _Discord
+bot_ for the _Monster Hunter World_ Video Game. This document specifically focuses on the 
+description, of the design process, for the following requirements. 
+
+**REQ 130** 
+
+_The bot shall be able to provide the recipe for the craft-able item requested by the user_
+
+
+**REQ 140**
+
+_The bot shall allow privileged users add data to the database_
+
+
+**REQ 150**
+
+
+_The bot shall allow privileged users edit data to the database_
+
+
+
+**REQ 160**
+
+_The bot shall allow users to track the monsters they have killed_
+
+
+**REQ 170**
+
+_The bot shall allow users to track the monsters that have killed them_
+
+
+**REQ 180**
+
+_The bot shall be able to display a list of all available commands supported by the bot_
+ 
+
+------------------------------------------------------------------------------------------
+
+
+######REQ 130 Design: 
```

Comment:
Please move this entire design section above the "Backend" section, so all of the REQs are together and in order.

Reply:
Understood